0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -36,67 +36,34 @@ The server used on the recipient's side to handle SPSP requests.
 
 #### Receiver
 
-The specific HTTP endpoint on the SPSP used for setting up a payment.
+The specific HTTPS endpoint on the SPSP used for setting up a payment.
 
 ## Overview
 
 ### Relation to Other Protocols
 
-SPSP is used for exchanging payment information before an ILP payment is initiated. It uses the receiver generated [Interledger Payment Requests](../0011-interledger-payment-request) over HTTP.
+SPSP is used for exchanging payment information before an ILP payment is initiated. The sender and recipient use the [Pre-Shared Key transport protocol(../0016-pre-shared-key/0016-pre-shared-key.md) to generate the condition and fulfillment. The recipient generates the shared secret to be used in PSK and communicates it to the sender over HTTPS.
```

Comment:
SPSP is totally tied to PSK. We realized that the combination of HTTPS and receiver-generated conditions would not give you non-repudiability anyway.

To get non-repudiability, you need the receiver to sign an IPR using a well-known key, such that you can show the signed payment request and corresponding fulfillment to other people and they can verify the proof that you fulfilled your debt to the receiver.

While it's totally possible to design an application layer protocol like this (and we might make one in the not so distant future), it's unnecessary for common use cases and it's definitely not simple. For normal payments, depending on the security of TLS is fine, and having proof that the recipient got the money (even if it's not proof you could convince other people with) is more than enough.

With PSK, getting the fulfillment from the transfer being executed is proof enough that the recipient got the money as long as a) the receiver didn't give the secret to anyone else (which would be stupid because then the receiver wouldn't get the money) b) the TLS connection was not man-in-the-middled c) the sender didn't use the secret to fulfill the transfer.